Sources/FluentPostgreSQL/PostgreSQLDriver.swift

```diff
@@ -2,7 +2,9 @@ import Fluent
 import PostgreSQL
 
 public class PostgreSQLDriver: Fluent.Driver {
-    public var idKey: String = "id"
+    
+    internal static let idKey: String = "id"
+    public var idKey: String = PostgreSQLDriver.idKey
```

Comment:
I added the `internal static let idKey` here because adding a new init argument in PostgreSQLDriver which inherits from GeneralSQLSerializer would necessitate still keeping an `init(sql:…)` where some hard coded idKey value would need to be placed. I think it's reasonable anyway that there is effectively a coupling here between a PostgreSQL serializer and driver (it's unlikely that the need to use the PostgreSQL serializer without the corresponding driver would arise).